app/views/admin/variant_overrides/_products_variants.html.haml

```diff
@@ -10,7 +10,8 @@
   %td.on_hand{ ng: { show: 'columns.on_hand.visible' } }
     %input{name: 'variant-overrides-{{ variant.id }}-count_on_hand', type: 'text', ng: {model: 'variantOverrides[hub_id][variant.id].count_on_hand'}, placeholder: '{{ variant.on_hand }}', 'ofn-track-variant-override' => 'count_on_hand'}
   %td.on_demand{ ng: { show: 'columns.on_demand.visible' } }
-    %input.field{ :type => 'checkbox', name: 'variant-overrides-{{ variant.id }}-on_demand', ng: { model: 'variantOverrides[hub_id][variant.id].on_demand' }, 'ofn-track-variant-override' => 'on_demand' }
+    %select{ name: 'variant-overrides-{{ variant.id }}-on_demand', ng: { model: 'variantOverrides[hub_id][variant.id].on_demand', options: 'option.value as option.description for option in onDemandOptions' }, 'ofn-track-variant-override' => 'on_demand' }
+      %option{ value: '' }
```

Comment:
has this empty value been decided with product people? maybe we should give them a chance to chose between this "" or "Use Producer Settings" as you have suggested initially.

Reply:
This was a last-minute decision I made while implementing when I realized we didn't need to label it, also in consideration that we might change the UI again for #3067.

But now, proposing the following much simpler set of options (should have thought of this earlier :grin:):

* Blank
* Yes
* No

Please see [my comment](https://github.com/openfoodfoundation/openfoodnetwork/issues/3066#issuecomment-440847944) in the issue.

Reply:
@Matt-Yorkley :point_up: This is related also to your feedback.